luigi/contrib/redshift.py

```diff
@@ -338,6 +401,9 @@ def run(self):
         self.copy(cursor, path)
         self.post_copy(cursor)
 
+        if self.do_enable_metadata_columns:
+            self.post_copy_metacolumns(cursor)
```

Comment:
I'm a bit confused on how this would interact with `post_copy`. Seems like you would never run both together.

Reply:
I've highly an example at the PR level displaying a use-case where we have both cases that run.